File: sdk-dist/python/src/examples/file_example.py
import os
from src.etherdata_sdk.file import File, FileObject

file = File(url=os.getenv("url"))

file_id = "1e000000000467d5d61db366e019da2e680b632c62a1fec85f159b3e91bd496ddcbbbb32ed31990caa23c5dd53f42ba10ef7b1946d11d705a96a444c1ae978b3"
file.download_file(file_id, download_path="./test.png")
print(file_id)

File: sdk-dist/typescript/lib/examples/file_example.ts
import { NodeFile, NodeFileObject } from "../file";

export async function uploadFile() {
  const fileAPI = new NodeFile(process.env.URL!);
  const file = new NodeFileObject({ filePath: "readme.md", days: 3 });
  const id = await fileAPI.uploadFile(file);
  return id;
}

export async function downloadFile(id: string) {
  const fileAPI = new NodeFile(process.env.URL!);
  await fileAPI.downloadFile({ fileId: id, downloadPath: "./test.md" });
  console.log(id);
}

(async () => {
  const id = await uploadFile();
  await downloadFile(id);
})();
